### DISCUSSION TITLE
Git sync should not sync environment api_keys

### DISCUSSION THREAD

**Discussion by DLittle123** [Kong/insomnia#6638] [Ideas]
Using the git sync for a collection seems to upload all the api_keys to a repository in the environment configuration.

Is there a way to turn that off so we do not commit api_keys to github?

**Comment by subnetmarco** [comment]
You can create private sub-environments which are never shared with Git or in the Cloud (when Cloud Sync is enabled):

<img width="611" alt="Screenshot 2023-10-24 at 9 34 36 AM" src="https://github.com/Kong/insomnia/assets/964813/c6e298f4-6391-4f0b-95e4-e9d33c1d64d1">

Therefore - ideally - you would have all synchable environment variable in the parent base environment, and secret keys stored in private sub environments.
